longevity_test.py

```diff
@@ -242,11 +241,13 @@ def test_custom_time(self):
                             self.log.debug('Stress cmd: {}'.format(stress_cmd))
                             self._run_all_stress_cmds(stress_queue, params)
 
-        fullscan = self._get_fullscan_params()
-        if fullscan:
-            self.log.info('Fullscan target: {} Fullscan interval: {}'.format(fullscan['ks.cf'],
-                                                                             fullscan['interval']))
-            self.run_fullscan_thread(ks_cf=fullscan['ks.cf'], interval=fullscan['interval'])
+        fs_params = self._get_read_operation_params(read_operation='run_fullscan')
```

Comment:
i think it is better to use a "full" meaningful name.. what about `full_scan_params`?

Reply:
fixed